parlai/crowdsourcing/tasks/acute_eval/acute_eval_blueprint.py

```diff
@@ -114,6 +118,13 @@ def assert_task_args(cls, args: DictConfig, shared_state: SharedTaskState) -> No
                     "Must provide `block_qualification` to use `block_on_onboarding_fail`"
                 )
 
+        if args.task.get('maximum_units_per_worker', None) != 1:
+            warn_once(
+                'It is *strongly* recommended to use a '
+                'args.task.maximum_units_per_worker value of 1, as was done in the '
+                'ACUTE-Eval paper! Go to GitHub Discussions for more information.'
```

Comment:
A slight change suggestion:
```
'It is *strongly* recommended to use a '
'args.task.maximum_units_per_worker value of 1, as was done in the '
'ACUTE-Eval paper! Anything else will not provide consistent results as '
'quoted there. Go to ParlAI's GitHub Discussion page '
'if you'd like to discuss further.
``` 
I think the current wording implies we already _have_ a discussion on it there.

Reply:
Good point, changed